lib/modules/manager/quadlet/extract.ts

```diff
@@ -0,0 +1,79 @@
+import { logger } from '../../../logger';
+import { getDep } from '../dockerfile/extract';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { QuadletFile } from './schema';
+
+function startsWithAny(image: string, prefixes: string[]): boolean {
+  return !!prefixes.find((prefix) => image.startsWith(prefix));
+}
+
+function endsWithAny(image: string, suffixes: string[]): boolean {
+  return !!suffixes.find((suffix) => image.endsWith(suffix));
+}
+
+function getQuadletImage(
+  image: string | null | undefined,
+  ignoredSuffixes: string[],
+  deps: PackageDependency<Record<string, any>>[],
+  config: ExtractConfig,
+): void {
+  if (
+    image &&
+    // Ignore transports that do not resolve to a registry
+    // https://github.com/containers/image/blob/main/docs/containers-transports.5.md
+    !startsWithAny(image, [
+      'dir:',
+      'docker-archive:',
+      'oci-archive:',
+      'oci:',
+      'containers-storage:',
+      'sif:',
+    ]) &&
+    !endsWithAny(image, ignoredSuffixes)
+  ) {
+    // Remove the docker:// or docker-daemon: transport prefix, if present
+    const cleanedImage = image
+      .replace(/^docker:\/\//, '')
+      .replace(/^docker-daemon:/, '');
```

Comment:
```suggestion
      .replace(regEx(/^docker:\/\//), '')
      .replace(regEx(/^docker-daemon:/), '');
```

use our regex helper